src/components/BorrowerProfile/LendersAndTeams.vue

```diff
@@ -30,29 +33,22 @@
 						class="tw-rounded md:tw-block tw-hidden tw-mr-2" :style="{width: '88px', height: '88px'}"
 					/>
 					<div :style="{display: 'block', height: 'auto', width: '50%'}">
-						<kv-loading-placeholder class="tw-mb-1 tw-w-full" :style="{height: '14px'}" />
+						<kv-loading-placeholder class="tw-mb-1 tw-flex-shrink-0 tw-w-full" :style="{height: '14px'}" />
 						<kv-loading-placeholder :style="{width: 40 + (Math.random() * 15) + '%', height: '14px'}" />
 					</div>
 				</div>
 			</div>
 
-			<div v-if="!loading" class="section-items tw-mb-3 tw-grid tw-grid-cols-2 tw-gap-2">
-				<!-- Replace with Generic component for showing Either Team or Lender image and information -->
-				<div
+			<div v-if="!loading" class="section-items tw-grid tw-grid-cols-2 tw-gap-2 tw-mb-3">
+				<supporter-details
 					v-for="(item, index) in truncatedItemList" :key="index"
-					class="tw-flex tw-flex-col md:tw-flex-row md:tw-items-center tw-mb-3"
-				>
-					<kv-loading-placeholder class="tw-rounded md:tw-hidden tw-mb-1.5" :style="{height: '128px'}" />
-					<kv-loading-placeholder
-						class="tw-rounded md:tw-block tw-hidden tw-mr-2" :style="{width: '88px', height: '88px'}"
-					/>
-					<div>
-						<span v-if="item.name">{{ item.name }}</span><br>
-						<span v-if="item.lenderPage && item.lenderPage.whereabouts">
-							{{ item.lenderPage.whereabouts }}
-						</span>
-					</div>
-				</div>
+					class="tw-flex tw-flex-col md:tw-flex-row md:tw-items-center"
```

Comment:
I'd suggest putting these layout classes inside the supporter-details component since I assume anytime you use supporter-details they would be desired.